qiskit/algorithms/time_evolvers/pvqd/pvqd_result.py

```diff
@@ -26,13 +24,13 @@ class PVQDResult(EvolutionResult):
 
     def __init__(
         self,
-        evolved_state: Union[StateFn, QuantumCircuit, OperatorBase],
-        aux_ops_evaluated: Optional[List[Tuple[complex, complex]]] = None,
-        times: Optional[List[float]] = None,
-        parameters: Optional[List[np.ndarray]] = None,
-        fidelities: Optional[List[float]] = None,
-        estimated_error: Optional[float] = None,
-        observables: Optional[List[List[float]]] = None,
+        evolved_state: QuantumCircuit,
+        aux_ops_evaluated: list[tuple[complex, complex]] | None = None,
+        times: list[float] | None = None,
+        parameters: list[np.ndarray] | None = None,
+        fidelities: list[float] | None = None,
+        estimated_error: float | None = None,
+        observables: list[list[float]] | None = None,
```

Comment:
I guess we have aux_ops_evaluated here and observables. See earlier comment on observables term in messages since we have no obervables in problem. Maybe the text here in the result correlates things a little but for me terminology consistent wth the problem would help - and observables earlier referred to operators here its more the expectation values I think. Its nothing changed by this PR but in looking over things again I guess I noticed more.